doc/tools/make_rst.py

```diff
@@ -85,14 +83,21 @@ def to_rst(self, state):  # type: ("State") -> str
             return make_type(self.type_name, state)
 
     @classmethod
-    def from_element(cls, element):  # type: (ET.Element) -> "TypeName"
+    def from_element(cls, element: ET.Element) -> "TypeName":
```

Comment:
Same question here for the stringly typing.

Reply:
This one refers to itself (class _TypeName_), so as far as I can tell this is the only way to do so. (Note that I am far from a Python expert, so if anyone has insights, please feel free to chime in.)